openseeds/ols-5/cohort-emails/README.md

```diff
@@ -0,0 +1,3 @@
+# Cohort emails
```

Comment:
Should we move that in the cohort GitHub repo together with cohort notes?

Reply:
Yeah, that seems easier to keep track of.